apps/assisted_tagging/filters.py

```diff
@@ -1 +1,31 @@
-#
+import django_filters
+
+from deep.filter_set import generate_type_for_filter_set
+from .models import DraftEntry
+from utils.graphene.filters import IDListFilter, MultipleInputFilter
+from .enums import (
+    DraftEntryTypeEnum
+)
+
+
+class DraftEntryFilterSet(django_filters.FilterSet):
+    lead = IDListFilter()
```

Comment:
We can remove filter_lead function with this

Let's also add basic tests for filters.

```suggestion
    leads = IDListFilter(field_name='lead')
```